modules/rhcos-environment-variables.adoc

```diff
@@ -29,11 +29,14 @@ You can use the following environment variables to configure {op-system} node sc
 |====
 |Environment Variable|Description
 
+|ROX_NODE_INDEX_ENABLED
+|Controls whether node indexing is enabled for this cluster. The default value is `false`. This variable is used for Scanner V4.
```

Comment:
> This variable is used for Scanner V4.

This confuses me a bit. It makes it sounds like we should set this variable in Scanner V4. This is set in the `compliance` container in the Collector DaemonSet, right? @Maddosaurus

How does this sound:

Set this variable to use Scanner V4-based RHCOS node scanning.